.ci/scripts/distribution/qa-testbench.sh

```diff
@@ -20,7 +19,7 @@ jobKey=""
 
 while true; do
     # activate jobs non-zero exits can be ignored with `<command> || true`
-    "${zbctl}" activate jobs "$businessKey" > activationresponse.txt 2>/dev/null || true
+    zbctl activate jobs "$businessKey" > activationresponse.txt 2>/dev/null || true
```

Comment:
Does this work? I would expect we need ./zbctl

Reply:
fixed, my test run failed because of this :)